### PR TITLE
Feature/ufs 273 questions

### PR DESCRIPTION
**Pull request by torchiad**

### CODE REVIEW

File: src/components/AuthWrapper/AuthWrapper.tsx [hunk 1]
```diff
@@ -42,6 +43,7 @@ export const AuthWrapper: FC<Props> = props => {
                 <ConfirmSignUp authProps={props} path="/confirm/:code" />
 
                 <ApplicationPage path="/application/:id" />
+                <QuestionPage path="/question/:id" />
```
Comment (ascorbic) [inline src/components/AuthWrapper/AuthWrapper.tsx]: Don't put this in this component. This is for wrapping the authentication components.

File: src/components/Question/Question.tsx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+            );
+        },
+        [
+            formState.values.applicationCase,
+            formState.values.complete,
+            updateFundingApplicationQuestion
+        ]
+    );
+
+    function onInputChange(event: string) {
```
Comment (ascorbic) [inline src/components/Question/Question.tsx]: Wrap this is useCallback.

File: src/components/Question/Question.tsx [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+                <Title mb={7}>Case for support</Title>
+
+                <Caption size="M" mb={1}>
+                    Provide your case for support
+                </Caption>
+                <Details
+                    mb={3}
+                    summary="What should I include in case for support section?"
+                >
+                    ...
```
Comment (ascorbic) [inline src/components/Question/Question.tsx]: The question hint goes here.

File: src/components/Question/Question.tsx [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+                </Details>
+                <form onSubmit={onSubmit}>
+                    <FormGroup error={!validForm}>
+                        {!validForm && (
+                            <ErrorText>
+                                {" "}
```
Comment (ascorbic) [inline src/components/Question/Question.tsx]: Remove the {" "}s

File: src/components/Question/Question.tsx [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+                        )}
+                        <TextArea
+                            {...textarea({
+                                name: "applicationCase",
+                                validate: onInputChange
+                            })}
+                        />
+                    </FormGroup>
+
+                    <div>Words remaining: {wordsRemaining}</div>
```
Comment (ascorbic) [inline src/components/Question/Question.tsx]: What's the div for? Use a govuk component for all text, or it doesn't get fonts applied.

File: src/components/QuestionPage/QuestionPage.tsx [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+            variables: {
+                id: props.id
+            },
+            fetchPolicy: "cache-first"
+        }
+    );
+
+    console.log(data);
+
+    const v: GetFundingApplicationQuestionQuery = {
```
Comment (ascorbic) [inline src/components/QuestionPage/QuestionPage.tsx]: Don't use mock data in the live page. That's what the storybook is for.

**Review by ascorbic — CHANGES_REQUESTED** [review]

File: src/components/AuthWrapper/AuthWrapper.tsx (hunk 1 above)
Comment (torchiad) [reply src/components/AuthWrapper/AuthWrapper.tsx]: Where would I put this? I tried initially in App.tsx, following the pattern on the other projects, however the page will load and then always be redirected back to "/"

Comment (torchiad) [reply src/components/AuthWrapper/AuthWrapper.tsx]: Fixed

**Comment by torchiad** [comment]
@ascorbic these items above have been fixed.

**Review by ascorbic — APPROVED** [review]

**Pull request merged**